tests/runtests.py

```diff
@@ -20,6 +20,13 @@
 from django.utils.deprecation import RemovedInDjango30Warning
 from django.utils.log import DEFAULT_LOGGING
 
+try:
+    import MySQLdb
+    # Ignore MySQL informational warnings that are triggered with queryset.explain()
+    warnings.filterwarnings('ignore', '\(1003, *', category=MySQLdb.Warning)
+except ImportError:
+    pass
```

Comment:
This would be better styled as:
```python
try:
    import MySQLdb
except ImportError:
    pass
else:
    # ...
    warnings.filterwarnings(...)
```